I.write-tests/05-get-second-third-smallest/get-second-third.test.js

```diff
@@ -1,3 +1,11 @@
+var getSecThird = require('./get-second-third')
+test('Return second and third item from an array and check original array', () => {
+    const expecion1 = [6, 8, [90, 5, 11, 8, 6]]
+    const exeption2 = [90, 5, 11, 8, 6]
+    const result = getSecThird(exeption2)
+    expect(result).toEqual(expecion1)
```

Comment:
The first array is supposed to be flat(no arrays inside it), the second one should have only the expected result [6,8].
You would then test the result against the second one.